### ISSUE TITLE
Bug: Swatches not working

### ISSUE THREAD

**Issue by CaoHavan** [Simonwep/pickr#58]
#### Do you want to request a **feature** or report a **bug**? 
Bug:

#### What is the current behavior?
i added option swatches, but it is not working. it is not show in color-picker. hmmm
my code:
```
const pickr = new Pickr({
            el: '.color-picker',

            default: '#42445A',

            swatches: [
                '#F44336',
                '#E91E63',
                '#9C27B0',
                '#673AB7',
                '#3F51B5',
                '#2196F3',
                '#03A9F4',
                '#00BCD4',
                '#009688',
                '#4CAF50',
                '#8BC34A',
                '#CDDC39',
                '#FFEB3B',
                '#FFC107'
            ],

            components: {

                preview: true,
                opacity: true,
                hue: true,

                interaction: {
                    hex: true,
                    rgba: true,
                    hsva: true,
                    input: true,
                    clear: true,
                    save: true
                }
            }
        });
```
#### Your environment:
```
Pickr-version (see Pickr.version):
Browser-version:  
Operating-system:  
```

**Comment by CaoHavan** [comment]
i founded. if i run code install by npm, swatches is not working. but i run code cloned from project, it working @@

**Comment by Simonwep** [comment]
Yes, the REAMDE is always on the level of the latest commit. Moreover, color swatches are currently in no release (but soon).

**Comment by Simonwep** [comment]
I've recently published [0.3.3](https://github.com/Simonwep/pickr/releases/tag/0.3.3) - now swatches should also available via npm.
